src/lib/ContractParser.js

```diff
@@ -14,6 +14,17 @@ import {
   soliditySignature
 } from './utils'
 
+function mapInterfacesToERCs (interfaces) {
+  interfaces = Object.keys(interfaces)
+    .filter(k => interfaces[k] === true)
+    .map(t => contractsInterfaces[t] || t)
+  return interfaces
+}
+
+function hasMethodSelector (txInputData, selector) {
+  return (selector && txInputData) ? txInputData.includes(selector) : null
```

Comment:
```suggestion
  return selector && txInputData && txInputData.includes(selector)
```

It is used only only to filter, right?